control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -3126,8 +3126,21 @@ func (r *HostedControlPlaneReconciler) reconcileOpenShiftAPIServer(ctx context.C
 		}
 	}
 
+	var (
```

Comment:
Why do we need the `var` block at all, could we just pass `hcp.Spec.AdditionalTrustBundle, hcp.Spec.Configuration`  to `oapi.ReconcileDeployment()`?

Reply:
yep, you're right.